include/zephyr/bluetooth/classic/avrcp.h

```diff
@@ -24,6 +24,13 @@ struct bt_avrcp_unit_info_rsp {
 	uint32_t company_id;
 };
 
+struct bt_avrcp_subunit_info_rsp {
+	uint8_t subunit_type;
+	uint8_t max_subunit_id;
+	uint8_t *extended_subunit_type;
+	uint8_t *extended_subunit_id;
```

Comment:
Should these be `const uint8_t *` to make it clear that the callback shouldn't try to modify the data?

Reply:
Will do.